masonry/src/passes/accessibility.rs

```diff
@@ -34,29 +34,28 @@ fn build_accessibility_tree(
             widget.item.short_type_name(),
             id.to_raw(),
         );
-        let current_node = build_access_node(widget.item, state.item, scale_factor);
+        let mut builder = build_access_node(widget.item, state.item, scale_factor);
 
         let mut ctx = AccessCtx {
             global_state,
             widget_state: state.item,
             widget_state_children: state.children.reborrow_mut(),
             widget_children: widget.children.reborrow_mut(),
             tree_update,
-            current_node,
             rebuild_all,
             scale_factor,
         };
-        set_common_properties(&mut ctx);
-        widget.item.accessibility(&mut ctx);
+        set_common_properties(&mut ctx, &mut builder);
```

Comment:
I think that `set_common_properties` and `build_access_node` can now be recombined into one function:
```
fn build_access_node(widget: &dyn Widget, ctx: &mut AccessCtx) -> NodeBuilder {...}
```